buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/publish/PublishingPlugin.groovy

```diff
@@ -102,6 +105,10 @@ class PublishingPlugin implements Plugin<Project> {
                     publications {
                         mavenAar(MavenPublication) {
                             from sub.components.android
+                            if (sub.name == 'ktx') {
```

Comment:
Why does this need to be explicitly set ? Can we not establish this build hierarchy in the respective gradle files ?

Reply:
It has to be here for now, as we are configuring all subprojects from the root project, I want to move a lot of this logic into FirebaseLibraryPlugin, but until then I added a todo here